### DISCUSSION TITLE
Behavior when no targets are specified

### DISCUSSION THREAD

**Discussion by jopemachine** [babel/babel#14533] [Q&A] [answered]
Documents say below statement in this document https://babeljs.io/docs/en/options#no-targets.

> When no targets are specified: Babel will assume you are targeting the oldest browsers possible. 
> For example, @babel/preset-env will transform all ES2015-ES2020 code to be ES5 compatible.

I checked when using `preset-env` config, the module is transpiled into CJS like the statement.

But when I'm trying to build an ES module with only `preset-react` config like below file, 

https://github.com/sindresorhus/fkill-cli/blob/f21d0be1741906d67c9581dcab1d2936f13dd9e9/package.json

It does not transpile the ESM to CJS unlike the `preset-env`'s case.

I would be glad to get to know related behaviors of Babel.

**Comment by nicolo-ribaudo** [comment, marked as answer]
"ESM to CJS" is not affected by the targets option, because it completely changes the semantics of your file. The conversion is handled by preset-enc, and you can control it using its "modules" option.
